skada/deep/base.py

```diff
@@ -253,17 +253,26 @@ def forward(self, X, sample_domain=None, is_fit=False, return_features=False):
             if self.domain_classifier_ is not None:
                 domain_pred_s = self.domain_classifier_(features_s)
                 domain_pred_t = self.domain_classifier_(features_t)
-                domain_pred = torch.empty((len(sample_domain)))
+                domain_pred = torch.empty(
+                    (len(sample_domain)), 
+                    device=domain_pred_s.device
+                )
                 domain_pred[source_idx] = domain_pred_s
                 domain_pred[~source_idx] = domain_pred_t
             else:
                 domain_pred = None
 
-            y_pred = torch.empty((len(sample_domain), y_pred_s.shape[1]))
+            y_pred = torch.empty(
+                (len(sample_domain), y_pred_s.shape[1]),
+                device = y_pred_s.device
```

Comment:
this is a pep8 violation. How come this is not caught by CIs?

Reply:
I don't know about the CI, but I fixed it in another PR anyway, thanks.